### DISCUSSION TITLE
MAUI Deployment Strategies

### DISCUSSION THREAD

**Discussion by a98c14** [dotnet/maui#6763] [Q&A]
To publish our WPF apps we are currently using  ClickOnce. Is there any equivalent deployment system for MAUI? Or what is everyone using to publish their apps and manage updates?

**Comment by Xyncgas** [comment]
MSIX is currently inside official documentation :https://docs.microsoft.com/en-us/dotnet/maui/windows/deployment/overview

**Comment by TonyK-FmGroup** [comment]
Is there, or will there be a way to deploy on Windows as a self contained EXE that does not require installing?
